src/components/Spacer/Spacer.scss

```diff
@@ -25,4 +25,21 @@
   --NegativeSpace-16: -16px;
   --NegativeSpace-8: -8px;
   --NegativeSpace-4: -4px;
+
+  --UncommonSpace-2: 2px;
```

Comment:
Hrm, I'm not sure I'm a fan of this many of these and/or if it'd just be better to leave it hard coded with the `12px` places cc @zach-ethos any thoughts?